charts/cni-metrics-helper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: cni-metrics-helper
-version: 1.16.0
+version: 1.16.1
```

Comment:
nit: you can remove this change, as this gets updated automatically when a new release is cut